pkgs/dart_mcp_server/README.md

```diff
@@ -150,6 +150,7 @@ For more information, see the official VS Code documentation for
 | `dart_format` | Dart format | Runs `dart format .` for the given project roots. |
 | `run_tests` | Run tests | Run Dart or Flutter tests with an agent centric UX. ALWAYS use instead of `dart test` or `flutter test` shell commands. |
 | `create_project` | Create project | Creates a new Dart or Flutter project. |
+| `build_runner` | Run build_runner | Runs `dart run build_runner build` for the given project roots. For this to succeed, the build_runner package must be a dev dependency of the package it is run on. |
```

Comment:
I am not necessarily against this additional tool, but we need to have some set of core-principles to figure out questions that are being posed below ahead of time.